components/bandwidth/sources/new-incoming-sms/new-incoming-sms.js

```diff
@@ -0,0 +1,23 @@
+module.exports = {
+  name: "New Incoming SMS",
+  description: "Emits an event each time a `message-received` event is received at the source url",
+  props: {
+    http: {
+      type: "$.interface.http",
+      customResponse: true,
+    },
+  },
+
+  async run(event) {
+    let messageBody = event.body[0];
```

Comment:
Small style comment: prefer `const` to `let` if you don't need to re-assignt the value later.

Reply:
Also, I'd recommend checking to make sure `messageBody` is defined prior to moving on:

```javascript
if (!messageBody) {
  return
}
```

Does Bandwidth send signatures with its webhook requests that can be validated here?